.github/workflows/build-release.yml

```diff
@@ -82,13 +84,18 @@ jobs:
           command: build
           args: --all --release --locked ${{ matrix.build_flags }}
 
+      - name: Get the version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF#refs/tags/}
+        shell: bash
+
       - name: Create archive
         run: |
           mkdir -p release/rage
           mv target/${{ matrix.target }}/release/rage* release/rage/
           rm release/rage/*.d
           tar czf ${{ matrix.archive_name }} -C release/ rage/
-        if: matrix.name != 'windows'
+        if: (matrix.name != 'windows' && startsWith(steps.get_version.outputs.VERSION ,'v'))
```

Comment:
I want to test these archiving steps as well. The intent of #93 is that every step in the release worflow runs except for "Upload archive to release".